examples/standalone/lrauv_control/lrauv_control.py

```diff
@@ -0,0 +1,199 @@
+#
+# Copyright (C) 2023 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+
+#
+# Check README for detailed instructions.
+# Usage:
+#   $ gz sim -r worlds/lrauv_control_demo.sdf
+#   $ # ./lrauv_control speed_m_s yaw_rad pitch_rad
```

Comment:
Should this line be `python3 lrauv_control.py speed_m_s yaw_rad pitch_rad`?

Reply:
Yep, you are right. Done in https://github.com/gazebosim/gz-sim/pull/2044/commits/80ff6f73b17428a77a26a68a414a145ea7f793c4.